app/src/main/res/layout/nav_header_navigation_drawer.xml

```diff
@@ -51,6 +51,7 @@
       android:fontFamily="sans-serif"
       android:text="@{@plurals/completed_story_count(viewModel.completedStoryCount, viewModel.completedStoryCount).concat(@string/bar_separator).concat(@plurals/ongoing_topic_count(viewModel.ongoingTopicCount, viewModel.ongoingTopicCount))}"
       android:textColor="@color/white"
-      android:textSize="14sp" />
+      android:textSize="14sp"
+      android:textAlignment="viewStart" />
```

Comment:
You can remove this change. It is not needed.

Reply:
On removing this change, the text shifts towards left in RTL view.

Reply:
Just to confirm - 
you are changing pseudo locale language to check the changes right

Reply:
Sorry @FareesHussain I did not get you

Reply:
@imraghav20 Check [this wiki](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines) to enable pseudo locale which helps to check RTL UI. Also, update screenshots by using the PseudoLocale.